docs/ref/contrib/admin/index.txt

```diff
@@ -1245,6 +1245,16 @@ subclass::
     a full count on the table which can be expensive if the table contains a
     large number of rows.
 
+.. attribute:: ModelAdmin.no_count
```

Comment:
The other flag is `show_full_result_count`. I think this one could also be made positive rather than negative, and also renamed to sit next to it in the docs and fit the naming scheme. How about `show_paginator_result_count` ?